### PR TITLE
upgrade css-modules-loader-core for Values support

### PR DESCRIPTION
**Pull request by joshwnj**
To be merged when https://github.com/css-modules/css-modules-loader-core/pull/28 is.

### CODE REVIEW

**Comment by tinchoz49** [comment]
:+1:

**Comment by bcomnes** [comment]
css-modules/css-modules-loader-core#28 is merged!  Is there anything left to do on this?

 :stuck_out_tongue_winking_eye:

File: package.json [hunk 1]
```diff
@@ -4,7 +4,8 @@
   "description": "A browserify transform to load CSS Modules",
   "main": "index.js",
   "dependencies": {
-    "css-modules-loader-core": "0.0.12",
+    "css-modules-loader-core": "^1.0.0-beta4",
```
Comment (bcomnes) [inline package.json]: This would just change to "^1.0.0"

**Comment by joshwnj** [comment]
Manually merged, and published to v0.13.0

**Comment by bcomnes** [comment]
\o/  I'm going to try to present css-modules at the next @pdxnode if I can get a slide deck together soon enough.

**Comment by joshwnj** [comment]
@bcomnes cool! Let us know how you go

**Pull request closed without merging**